cpmpy/solvers/pindakaas.py

```diff
@@ -0,0 +1,322 @@
+# /usr/bin/env python
+"""
+Interface to Pindakaas (`pkd`) API.
+
+Requires that the `pkd` library python package is installed:
+
+    $ pip install pindakaas
+
+`pkd` is a library to transform pseudo-Boolean and integer constraints into conjunctive normal form.
```

Comment:
please see the new way of documenting the top of solvers on master

(and; pdk?)